CondTools/SiPixel/test/BuildFile.xml

```diff
@@ -19,3 +19,9 @@
   </bin>
 </environment>
 
+<environment>
+  <bin   file="testPixelDBs.cpp">
```

Comment:
@tvami this new test used the same `testPixelDBs.cpp` file as the already existing test `run_0T_BoR3`.
Either split it (creating a new `testPixePhaselDBs.cpp`) or put everything in the same bash executable.